kubernetes/skaffold/src/test/kotlin/com/google/kubernetes/tools/skaffold/DefaultSkaffoldExecutorServiceTest.kt

```diff
@@ -230,40 +225,54 @@ class DefaultSkaffoldExecutorServiceTest {
         assertThat(result.commandLine).isEqualTo("skaffold dev --filename test.yaml")
     }
 
-    @Test
-    fun `isSkaffoldAvailable returns true when skaffold execution returns 0 exit code`() {
-        mockSkaffoldExecution()
-        every { mockProcess.exitValue() } answers { 0 }
-        assertThat(defaultSkaffoldExecutorService.isSkaffoldAvailable()).isTrue()
-    }
+    // skaffold init tests
 
     @Test
-    fun `isSkaffoldAvailable returns false when skaffold execution returns 1 exit code`() {
-        mockSkaffoldExecution()
-        every { mockProcess.exitValue() } answers { 1 }
-        assertThat(defaultSkaffoldExecutorService.isSkaffoldAvailable()).isFalse()
+    fun `skaffold init with analyze generates valid command line`() {
```

Comment:
what about a test for both `analyze` and `force`?

Reply:
the flags are mutually exclusive, and shouldn't be used together. I enforced that and added a new test case.